tests/mockLogzioListener/logsList.py

```diff
@@ -0,0 +1,9 @@
+# Modules imported only once, this is effectively a singleton
```

Comment:
Its being accessed from 2 different classes that I can't pass this as a parameter.
So I'm using the "import once" policy of python as a singleton. that is the only purpose